src/gvizdoom_client/App.cpp

```diff
@@ -66,6 +66,20 @@ void App::loop(void)
         return;
     }
 
+    std::vector<Action> actions;
+    {
+        for (size_t i = 0; i < 50; ++i)
+            actions.emplace_back(ACTION_ATTACK);
+
+        for (size_t i = 0; i < 50; ++i)
+            actions.emplace_back(static_cast<int>(ACTION_FORWARD | ACTION_ATTACK));
+
+        for (size_t i = 0; i < 50; ++i)
+            actions.emplace_back(ACTION_FORWARD);
+    }
```

Comment:
Is there a reason for the explicit scope?

Reply:
trying to communicate that these lines of code are related to each other, limiting the scope of possible other tempfoo code like this